build/common.sh

```diff
@@ -90,17 +90,48 @@ readonly RELEASE_STAGE="${LOCAL_OUTPUT_ROOT}/release-stage"
 readonly RELEASE_DIR="${LOCAL_OUTPUT_ROOT}/release-tars"
 readonly GCS_STAGE="${LOCAL_OUTPUT_ROOT}/gcs-stage"
 
-# The set of master binaries that run in Docker (on Linux)
+
+# Get the set of master binaries that run in Docker (on Linux)
 # Entry format is "<name-of-binary>,<base-image>".
 # Binaries are placed in /usr/local/bin inside the image.
-readonly KUBE_DOCKER_WRAPPED_BINARIES=(
-  kube-apiserver,busybox
-  kube-controller-manager,busybox
-  kube-scheduler,busybox
-  kube-proxy,gcr.io/google_containers/debian-iptables:v1
-)
+# 
+# $1 - server architecture
+kube::build::get_docker_wrapped_binaries() {
+  if [[ $1 == "amd64" ]]; then
```

Comment:
One simple reason, I haven't used it before :)
At least not that much.
I'll check the syntax...